test/e2e/k8s_util.go

```diff
@@ -746,22 +746,21 @@ func (k *KubernetesUtils) Bootstrap(namespaces, pods []string) (*map[string][]st
 	return &podIPs, nil
 }
 
-func (k *KubernetesUtils) Cleanup(namespaces []string) error {
+func (k *KubernetesUtils) Cleanup(namespaces []string) {
 	// Cleanup any cluster-scoped resources.
 	if err := k.CleanACNPs(); err != nil {
-		return err
+		log.Infof("Error when cleaning-up ACNPs: %v", err)
```

Comment:
Any reason it doesn't use `log.Errorf`?

Reply:
I didn't realize that this was `github.com/sirupsen/logrus` and that there was an `Errorf` method :)

Updated. I also updated `LegacyCleanup` to match `Cleanup`.